config/settings/local.py

```diff
@@ -9,6 +9,11 @@
 # https://docs.djangoproject.com/en/dev/ref/settings/#allowed-hosts
 ALLOWED_HOSTS = ["*"]
 
+TEMPLATE_DEBUG = env("TEMPLATE_DEBUG", default=False) == 1
+
+if TEMPLATE_DEBUG:
```

Comment:
You could squish this down to

```suggestion
if env("TEMPLATE_DEBUG", default=False) == 1:
```

if you want and it might be marginally more readable than the `... = ... == ...` form above, but no strong feelings.

Reply:
what is the expected truthy response, is it not True?